suite-common/wallet-core/src/device/deviceThunks.ts

```diff
@@ -30,14 +30,15 @@ import { PORTFOLIO_TRACKER_DEVICE_ID, portfolioTrackerDevice } from './deviceCon
  */
 export const selectDeviceThunk = createThunk(
     `${MODULE_PREFIX}/selectDevice`,
-    (device: Device | TrezorDevice | undefined, { dispatch, getState }) => {
+    (deviceId: Device['id'] | TrezorDevice['id'] | undefined, { dispatch, getState }) => {
```

Comment:
type can by simplified to either
```suggestion
    (deviceId: Device['id'], { dispatch, getState }) => {
```
or 
```suggestion
    (deviceId?: string | null, { dispatch, getState }) => {
```